CRM/Remoteevent/Upgrader.php

```diff
@@ -193,6 +193,22 @@ public function upgrade_0015()
         return true;
     }
 
+    /**
+     * Synchronizes custom fields for CiviRemote Event registration
+     * configuration fields, as two fields need to be marked as not searchable
+     * for avoiding exceeding index lengths with bigger VARCHAR lengths
+     * introduced with upgrade_0015().
+     *
+     * @return void
+     * @throws \Exception
+     */
+    public function upgrade_0016()
```

Comment:
I'd suggest to use PHP type hint instead of PHPDoc type for the return type.

Reply:
Done. Also, this made me see the missing `return true` for the upgrader, so thanks for nitpicking :wink: